### PR TITLE
Making changes

### PR DESCRIPTION
**Pull request by 1san1**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by furkanmutlu** [comment]
@1san1 is there any reason why did you create new PR ? If this is the is what you want to merge, please close previous one.

**Comment by furkanmutlu** [comment]
@1san1  Seems like you have merge conflict. Please pull latest changes from main branch

File: frontend/src/components/Gallery/Gallery.jsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
 import React, { useState } from "react";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faExpandAlt } from "@fortawesome/free-solid-svg-icons";
+
+import LikeUI from "../ImageLikeUI/LikeUI";
```
Comment (furkanmutlu) [inline frontend/src/components/Gallery/Gallery.jsx:4]:
```suggestion
```

File: frontend/src/components/Gallery/Gallery.jsx [hunk 2]
```diff
@@ -28,17 +40,9 @@ const Gallery = ({ images }) => {
...
-                      src={img.src}
-                      alt={img.title}
-                    />
-                    <button
-                      className="expand-button"
-                      onMouseDown={() => handleShow(img)}
-                    >
-                      <FontAwesomeIcon icon={faExpandAlt} inverse size="md" />
-                    </button>
+                    {/* here i used separate component called "LikeUI" */}
```
Comment (furkanmutlu) [inline frontend/src/components/Gallery/Gallery.jsx:43]: Why not adding like button here, instead of creating new component?

**Review by furkanmutlu — CHANGES_REQUESTED** [review]

File: frontend/src/components/Gallery/Gallery.jsx (hunk 2 above)
Comment (1san1) [reply frontend/src/components/Gallery/Gallery.jsx:43]: In order to make "likes" independent to that image only.

**Comment by furkanmutlu** [comment]
Closing this since I already covered it on  #54 . You may take a look at my solution. I also included storing likes on localstorage. It was actually more complex then I thought. Thank you for your help. And please keep an eye on issues here, you will see more issues coming. Or if you have an idea create one ☺️

**Pull request closed without merging**